storage/backup/backup-cli/src/backup.rs

```diff
@@ -77,8 +77,9 @@ pub async fn backup_account_state(
                 .write_new_file(stream::once(async move { chunk }))
```

Comment:
what is the "4" on line 74?

Reply:
it's the u32 indicating len of the record. (made explicit in new commit)